fagiClient/src/com/fagi/network/Communication.java

```diff
@@ -110,4 +117,6 @@ public void close() {
     public Exception getNextException() {
         return inputHandler.containsException();
     }
+
+    public void setEncryption(EncryptionAlgorithm encryption) { this.encryption = encryption; }
```

Comment:
Setting the encryption would fuck up everything if done on runtime. This looks like a dangerous public method which should be deleted.